staging/src/k8s.io/apiextensions-apiserver/third_party/forked/celopenapi/model/schemas.go

```diff
@@ -106,6 +123,11 @@ func SchemaDeclType(s *schema.Structural, isResourceRoot bool) *DeclType {
 		if s.ValueValidation != nil {
```

Comment:
What about plain strings? Shouldn't `return StringType` on line 138 return a StringType with a maxLength if it's provided?

Reply:
Gotcha, fixed